README.md

```diff
@@ -64,6 +65,13 @@ In order to get started quickly, you can deploy the docker image to a DigitalOce
 
 *Disclaimer*: The DigitalOcean server is publicly accessible on the Internet. Do not put sensitive information on the network that you would not want someone else to know. Anyone with access to the network will be able to use the root account above.
 
+### Enable Optional Soroban RPC server
+
+You can run an instance of Soroban RPC server in the container also. At present the Soroban RPC server can only be enabled when using standalone network `--standalone`. Add the following command line flags to the `docker run` container invocation to enable Soroban RPC server to be launched:
```

Comment:
Can you add a big warning here noting that the Soroban RPC server is in early development, and should not be used in production, and its API is not stable?

Some people use this quickstart image in production-like environments, or to run test network environments. It's important we steer these folks away from enabling it as it is very unstable at this point.